pkg/appStore/deployment/service/InstalledAppService.go

```diff
@@ -889,7 +932,7 @@ func (impl InstalledAppServiceImpl) GetInstalledAppVersionHistoryValues(installe
 	values.ValuesYaml = versionHistory.ValuesYamlRaw
 	return values, err
 }
-func (impl InstalledAppServiceImpl) FetchResourceTree(rctx context.Context, cn http.CloseNotifier, appDetail *bean2.AppDetailContainer) (bean2.AppDetailContainer, error) {
```

Comment:
why did you remove the error returning, logic behind it?

Reply:
not done by me